serverTest.test.js

```diff
@@ -0,0 +1,86 @@
+const request = require("supertest");
+const app = require("./app");
+const database = require("./DBClass");
+const fs = require("fs");
+
+afterAll(() => {
+  database.urlObject.urlArray.pop();
+  fs.writeFile(
+    `./data/test.json`,
+    JSON.stringify(
+      {
+        urlArray: [],
+      },
+      null,
+      4
+    ),
+    (err) => {
+      if (err) {
+        throw new Error("Cant find file:" + err);
+      }
+    }
+  );
+});
+
+describe("check post route to /api/shorturl/", () => {
+  it("should return original url and short url", async () => {
+    const response = await request(app)
+      .post("/api/shorturl/")
+      .send({ url: "https://www.youtube.com/feed/subscriptions" });
+
+    const url = response.body;
+    const originalUrl = url.originalUrl;
+    expect(response.status).toBe(200);
+    expect(originalUrl).toBe("https://www.youtube.com/feed/subscriptions");
+    expect(response.body["shortUrlId"]).toBeDefined;
```

Comment:
This is great, but you can even test further and check if the returned `shortUrlId` is valid. 
You can use regex, helper function, a library etc...